api/src/main/java/jakarta/mail/Header.java

```diff
@@ -67,4 +67,36 @@ public String getName() {
     public String getValue() {
 	return value;
     }
+
+    @Override
+    public int hashCode() {
+        final int prime = 31;
```

Comment:
Could `java.util.Objects.hash(name, value)` be used instead?

Reply:
Sure